internal/parse/parts_test.go

```diff
@@ -4,30 +4,37 @@ import (
 	"testing"
 
 	"github.com/canonical/sqlair/internal/parse"
-	"github.com/stretchr/testify/assert"
+	. "gopkg.in/check.v1"
 )
 
-func TestInputPart(t *testing.T) {
+// Hook up gocheck into the "go test" runner.
+func Test(t *testing.T) { TestingT(t) }
+
+type PartsSuite struct{}
+
+var _ = Suite(&PartsSuite{})
+
+func (s *PartsSuite) TestInputPart(c *C) {
 	i := parse.InputPart{
 		parse.FullName{
 			Prefix: "mytype",
 			Name:   "mytag",
 		},
 	}
-	assert.NotEqual(t, nil, i)
-	assert.Equal(t, "mytype", i.Source.Prefix)
-	assert.Equal(t, "mytag", i.Source.Name)
+	c.Assert(i, Not(IsNil))
```

Comment:
This is impossible to be nil unless the compiler is broken. Indeed, it's not even a pointer.

As an aside, IIRC we have NotNil, but typically we don't do this because we always test more details about the value than simply it being nil or not. If it was nil, the logic below will break, for instance.

Reply:
I think the original test didn't make sense either since this was not a pointer anyway. The rest of the tests looks good to me here and in `parser_test.go`